<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>901</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SWIsop2</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>VaultTalk</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QLineEdit" name="receiverInput">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="senderInput">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>390</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>420</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="messageInput">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="logOutButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsPage">
     <widget class="QPushButton" name="homeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
